lib/daru/io/io.rb

```diff
@@ -41,13 +41,15 @@ class << self
       def from_excel path, opts={}
```

Comment:
Actually [travis log](https://travis-ci.org/SciRuby/daru/jobs/567213389#L1653) says that single method should not have more than 15 lines of code. Since you added 2 more lines it is crossing the number of lines (which is 15).

You can check it manually as well by running the `rubocop` in the root directory. Please go through the development workflow and setup guide to understand it better.

Solution for this is create a separate method(break this from_excel method into pieces) to reduce the number of lines.